x11-libs/wxGTK/wxGTK-3.2.2.1-r4.ebuild

```diff
@@ -0,0 +1,257 @@
+		-e '/WX_VERSION_TAG=/ s:${WX_RELEASE}:3.0:' \
+		configure || die
+}
+
+multilib_src_configure() {
+	# Workaround for bug #915154
+	append-ldflags $(test-flags-CCLD -Wl,--undefined-version)
+
+	# X independent options
+	local myeconfargs=(
+		--with-zlib=sys
+		--with-expat=sys
+		--enable-compat30
+		--enable-xrc
+		$(use_with sdl)
+		$(use_with lzma liblzma)
+		# Currently defaults to curl, could change.  Watch the VDB!
+		$(use_enable curl webrequest)
+
+		# PCHes are unstable and are disabled in-tree where possible
+		# See bug #504204
+		# Commits 8c4774042b7fdfb08e525d8af4b7912f26a2fdce, fb809aeadee57ffa24591e60cfb41aecd4823090
+		$(use_enable pch precomp-headers)
+
+		# Don't hard-code libdir's prefix for wx-config
+		--libdir='${prefix}'/$(get_libdir)
+	)
+
+	# By default, we now build with the GLX GLCanvas because some software like
+	# PrusaSlicer does not yet support EGL:
+	#
+	# https://github.com/prusa3d/PrusaSlicer/issues/9774 .
+	#
+	# A solution for this is being developed upstream:
+	#
+	# https://github.com/wxWidgets/wxWidgets/issues/22325 .
+	#
+	# Any software that needs to use OpenGL under Wayland can be patched like
+	# this to run under xwayland:
+	#
+	# https://github.com/visualboyadvance-m/visualboyadvance-m/commit/aca206a721265366728222d025fec30ee500de82 .
+	#
+	# Check that the macro wxUSE_GLCANVAS_EGL is set to 1.
+	#
+	myeconfargs+=( "--disable-glcanvasegl" )
+
+	# debug in >=2.9
+	# there is no longer separate debug libraries (gtk2ud)
+	# wxDEBUG_LEVEL=1 is the default and we will leave it enabled
+	# wxDEBUG_LEVEL=2 enables assertions that have expensive runtime costs.
+	# apps can disable these features by building w/ -NDEBUG or wxDEBUG_LEVEL_0.
+	# http://docs.wxwidgets.org/3.0/overview_debugging.html
+	# https://groups.google.com/group/wx-dev/browse_thread/thread/c3c7e78d63d7777f/05dee25410052d9c
+	use debug && myeconfargs+=( --enable-debug=max )
+
+	# wxGTK options
+	#   --enable-graphics_ctx - needed for webkit, editra
+	#   --without-gnomevfs - bug #203389
+	use X && myeconfargs+=(
+		--enable-graphics_ctx
+		--with-gtkprint
+		--enable-gui
+		--with-gtk=3
+		--with-libpng=sys
+		--with-libjpeg=sys
+
+		# Choosing to enable this unconditionally seems fair, pcre2 is
+		# almost certain to be installed.
+		--with-regex=sys
+		--without-gnomevfs
+		$(use_enable gstreamer mediactrl)
+		$(multilib_native_use_enable webkit webview)
+		$(use_with libnotify)
+		$(use_with opengl)
+		$(use_with tiff libtiff sys)
+		$(use_enable keyring secretstore)
+		$(use_enable spell spellcheck)
+		$(use_enable test tests)
+		$(use_enable wayland)
+	)
+
+	# wxBase options
+	! use X && myeconfargs+=( --disable-gui )
+
+	# wxWidgets installs a configuration file with a reference to EGREP.
+	# Autoconf discovers these programs via full paths, which is
+	# unnecessary and fails if a build happened on a merged-usr system
+	# but is being used on a split-usr system.  Bug #927920.
+	export ac_cv_path_SED="$(basename "$(type -P sed)")"
```

Comment:
https://gitweb.gentoo.org/repo/gentoo.git/commit/dev-build/libtool?id=411f7e106012a6af37a5d14e2a62572c2f0b4f6a

Reply:
arsen, can you update it please?